anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -180,6 +223,30 @@ def update_feed_group_counts(self) -> None:
         """
         ...
 
+    def toggle_feed_enabled(
```

Comment:
(nit) This is not the most descriptive name. I know it's copied over. How about `update_feed_status` or something like it